tsconfig.json

```diff
@@ -1,12 +1,14 @@
 {
     "compilerOptions": {
+        "declaration": true,
```

Comment:
Is there a way to output all the type definitions into one file?

Reply:
According to docs the intention is for there to be separate .d.ts files to mirror the original .ts files. There's at least one plugin (https://api-extractor.com/) that can roll them up, but I don't think it's a big deal